packages/scripts/helpers/rollup-plugin-angularjs-template-loader.js

```diff
@@ -25,6 +25,8 @@ module.exports = function angularJsTemplateLoader(options = {}) {
 
       // Find the directory the JS source file is in
       const baseDir = fs.lstatSync(id).isDirectory() ? id : path.dirname(id);
+      const moduleRootDir = path.resolve('.');
```

Comment:
wondering how this will behave if you run e.g. `yarn build` from a subdirectory like `src/core/presentation` and not from the root

Reply:
```suggestion
      const deckRootDir = path.resolve(__dirname, '..', '..', '..');
      const modulesRootDir = path.resolve(deckRoot, 'app', 'scripts', 'modules');
```

Reply:
Good call. I haven't run `yarn` from a subdirectory and TIL that it works!.

Reply:
There's an issue with `path.resolve(__dirname, '..', '..', '..');` though. `__dirname` would point to the one within `node_modules`. For ex, it could be `deck/app/scripts/modules/amazon/node_modules/@spinnaker/scripts/helpers`.

Reply:
I ran a quick test. `path.resolve('.');` ends up resolving to the directory that has `package.json`. Wondering if `yarn` is doing some magic there.

Reply:
Cool as long as works 👍